parser-typechecker/src/Unison/Codebase/Editor/VersionParser.hs

```diff
@@ -12,12 +12,12 @@ import qualified Unison.Codebase.Path as Path
 import Data.Void (Void)
 
 -- |"release/M1j.2" -> "releases._M1j"
---  "devel/*" -> "trunk"
+--  "latest-*" -> "trunk"
 defaultBaseLib :: Parsec Void Text ReadRemoteNamespace
-defaultBaseLib = fmap makeNS $ devel <|> release
+defaultBaseLib = fmap makeNS $ latest <|> release
   where
-  devel, release, version :: Parsec Void Text Text
-  devel = "devel/" *> many anyChar *> eof $> "trunk"
+  latest, release, version :: Parsec Void Text Text
+  latest = "latest-" *> many anyChar *> eof $> "trunk"
```

Comment:
Okay, so this will use a released version of base if you're using a named release, and otherwise will pull trunk.

Reply:
> Looks great. I'd add the `--no-base` flag, but that could be a follow up.
> 
> If you're interested limiting the output of that initial pull, we could add `pull.quiet` command, which just produces `✅ Finished.` output. It could be implemented by adding an extra argument to the existing `PullRemoteBranchI` constructor, with verbosity info. And then dispatch on this in `HandleInput`. This would be a good way to learn about adding new commands to `ucm`.

Yeah, this sounds like something we'd be interested in!

Reply:
Yep. This was already being used with regards to the "pull base" hint, but had a bug with `devel`